hackclub.com.yaml

```diff
@@ -2380,8 +2380,12 @@ ru:
     value: cname.vercel-dns.com.
 runshaw:
   - ttl: 600
-    type: CNAME
-    value: dragon863.hackclub.app.
+    type: ALIAS
+    value: hackclub.app.
+  - ttl: 600
+    type: TXT
+    values:
+      - domain-verification=dragon863
```

Comment:
Please put this at the top of the file like the README says

Reply:
Sorry, I'm not sure I understand which part I should be moving? If you're referring to the txt record I don't think it should be under the vercel section as I'm trying to verify with nest as shown in [the nest wiki](https://guides.hackclub.app/index.php/Subdomains_and_Custom_Domains)

Reply:
this is what I get for reviewing on my phone 🤦‍♀️